components/atom/tag/src/Actionable/Container.js

```diff
@@ -4,9 +4,16 @@ import PropTypes from 'prop-types'
 /**
  * Component treated as an anchor when href is defined
  */
-const ActionableTagContainer = ({Link, href, target, children, ...props}) => {
+const ActionableTagContainer = ({
+  Link,
+  href,
+  target,
+  rel,
+  children,
+  ...props
```

Comment:
As props is being destructured, it is passing `rel` to <Link /> component as a prop